Update boardMapper.xml

aurevoir-jaeho opened the pull request:
판매글 목록 조회 xml코드 추가

aurevoir-jaeho committed:
Update boardMapper.xml

판매글 목록 조회 xml코드 추가

Lee-KS-kr reviewed (changes requested):
36번줄
soldout은 default 'N'이기 때문에 null체크를 할 필요는 없습니다.
<where>태그 빼고 그냥 where로 단일체크 해주세용

aurevoir-jaeho committed:
Update boardMapper.xml

aurevoir-jaeho committed:
Merge branch 'main' into KJH

aurevoir-jaeho committed:
Update boardMapper.xml

<!-- 판매글 목록 조회 -->
동적 쿼리 if문의 where 조건문
-> 단순 where 조건문으로 수정
이유 : SelectAll 이기 때문에

Lee-KS-kr reviewed (approved):
LGTM
